### ISSUE TITLE
[Feature Requst] Support for dataview rendering within callout blocks

### ISSUE THREAD

**Issue by yett1873** [oleeskild/obsidian-digital-garden#285]
Currently the plugin won't render dataview queries with `>` before them and returns an error message: 
```
> [!database]- 
> ```dataview
> TABLE date_0, date_1,abstract, comment, rating
> FROM #obj/hx AND #timeline
> SORT date_0 ASC
> ``` 
```
![image](https://user-images.githubusercontent.com/75569851/233546014-c26029fa-178d-45e1-b222-ea92e6dc8e6e.png)

I enclose most of my queries with callouts because collapsable callouts can be really useful when they return long results. It'd be really helpful if we could include this feature. Huge thanks!

**Comment by rokville** [comment]
+1 I have the same use-case and issue.

**Comment by Shubedobedo** [comment]
+1 Same here

**Comment by mafergon** [comment]
+1

**Comment by oleeskild** [comment]
This has been fixed in the latest release, v2.50.5. It is currently only fixed for dataview, and not dataviewjs blocks.
